lib/ansible/modules/cloud/vmware/vsphere_copy.py

```diff
@@ -206,7 +206,7 @@ def main():
         try:
             if isinstance(e[0], int):
                 error_code = e[0]
-        except (KeyError, TypeError):
+        except (KeyError, TypeError, IndexError):
```

Comment:
I warned for this, the catchall Exception at above is the wrong approach.

Reply:
@dagwieers I am totally agree with you. @lijok Could you please do the needful, thanks.